### PR TITLE
set objects

### PR DESCRIPTION
**Pull request by fschlimb**
gilectomy'zing set objects

### CODE REVIEW

**Comment by fschlimb** [comment]
@larryhastings

File: Modules/gcmodule.c [hunk 1]
```diff
@@ -1124,7 +1124,7 @@ invoke_gc_callback(const char *phase, int generation,
 static Py_ssize_t
 collect_with_callback(int generation)
 {
-    Py_ssize_t result, collected, uncollectable;
+    Py_ssize_t result = 0, collected = 0, uncollectable = 0;
```
Comment (larryhastings) [inline Modules/gcmodule.c:1127]: What does this have to do with set?

File: y.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```
Comment (larryhastings) [inline y.py:1]: While I approve of you checking in your test case, maybe you could fold it into Lib/test/test_set.py somewhere?

File: Modules/gcmodule.c (hunk 1 above)
Comment (fschlimb) [inline Modules/gcmodule.c:1127]: Nothing. It's only silencing a gcc warning. I can make it a separate PR if you like.

File: y.py (hunk 2 above)
Comment (fschlimb) [inline y.py:1]: Will do so here - later today.
